Makefile

```diff
@@ -112,6 +112,11 @@ travis: $(VENV_PATH)
 	$(MAKE) $(LIMBO_REQUIREMENTS)
 	env/bin/tox
 
+gh-tox:
```

Comment:
Since this will only ever be used from github, and not typed manually, I would suggest not abbreviating it to make it clearer (github-tox).

Reply:
ack, good idea!